Sources/Client/DeltaClientApp.swift

```diff
@@ -127,9 +132,17 @@ struct DeltaClientApp: App {
     Self.modalState.update(to: .error(message, safeState: safeState))
   }
   
-  /// Logs a fatal error and then fatal errors.
-  static func fatal(_ message: String) -> Never {
+  /// Logs a fatal error and redirects to troubleshooting page.
+  static func fatal(_ message: String) {
     log.critical(message)
-    fatalError(message)
+    loadingState.update(to: .done(nil)) // If loading, removes loading screen
```

Comment:
You should avoid having to set loading state to done with nil, that is misleading because it's not actually done. So if some code checks the state to see if loading is complete, it will think that loading is complete when it possibly isn't.

Reply:
On it 👍